packit/sync.py

```diff
@@ -144,13 +144,17 @@ def command(self, fail_on_missing: bool = False) -> list[str]:
         command += [self.dest]
         return command
 
-    def resolve(self, src_base: Path = Path.cwd(), dest_base: Path = Path.cwd()):
```

Comment:
oh fuck

Reply:
???

Reply:
it returns an object, so it returned the `cwd()` only first time /o and no one noticed it